### ISSUE TITLE
RTL problems on the edit lesson details page in coach

### ISSUE THREAD

**Issue by MisRob** [learningequality/kolibri#9068]
## Observed behavior

When I edit details of a lesson in coach, there are the following problems for RTL languages:

1. "Entire class" radio button not  right-aligned in the recipients section
2. "Entire class" and "Individual learners" icons are not right-aligned and overlap with labels in the recipients section
2. Missing space between a resource label and its icon in the resources section

| RTL | LTR (no problems, for reference only) |
| ----- | ----- |
| ![lesson details rtl](https://user-images.githubusercontent.com/13509191/151778957-586605a3-f694-49bf-8c07-c7090240e94d.png) | ![lesson details ltr](https://user-images.githubusercontent.com/13509191/151778998-77d38145-91e0-48b2-9d05-77184cf2406f.png) |

## Expected behavior

1. "Entire class" radio button is right-aligned in the recipients section
2. "Entire class" and "Individual learners" icons are right-aligned
3. There's a space between a resource label and its icon in the resources section

## User-facing consequences

Difficulties with locating controls and bad readability of labels

## Steps to reproduce

You need to have a lesson started.

1. As a coach, navigate to "Coach" --> "Reports" --> "Lessons"
2. Select a lesson
3. On the lesson page, click on the "Options" dropdown and select "Edit details"
4. See the "Edit lesson details" page

Note that there are more navigation paths to get to the "Edit lesson details" page and its content can vary according to the chosen path. Make sure to navigate through "Reports" --> "Lessons" to be able to preview the resources list on the page.

## Context

- Kolibri 0.15.1.dev0+git.20220127215827
- Ubuntu 20.04.3 LTS
- Chrome Version 97.0.4692.99 (Official Build) (64-bit)

**Comment by MisRob** [comment]
This might not be a problem, I forgot to check on it without hot reloading on. I will try it tomorrow again with hot reload off.

**Comment by MisRob** [comment]
Yes, the problem was on my side, I'm sorry @pcenov and @radinamatic.

It's fine with the dev server hot reload turned off indeed:
![Screenshot from 2022-02-02 14-45-07](https://user-images.githubusercontent.com/13509191/152165720-e406f9c7-ef5d-4ca6-827b-0725e5ef1983.png)
